manifests/kustomize/env/platform-agnostic/mysql/mysql-deployment.yaml

```diff
@@ -15,11 +15,15 @@ spec:
       labels:
         app: mysql
     spec:
+      serviceAccountName: mysql
       containers:
-      - env:
+      - args:
+        - --datadir
+        - /var/lib/mysql/datadir
```

Comment:
Is this a breaking change?

Reply:
It just makes the storage location more explicit and in line with other mysql deployments in the kubeflow namespaces e.g. https://github.com/kubeflow/manifests/blob/f4cca979d3020a04e45db65bc58d7bbfe386ffd6/contrib/metadata/overlays/db/metadata-db-deployment.yaml#L26 . To be honest i would have to retest, because it might also work with out, but it is good practice anyway.

Reply:
I took a quick look on my instance, it seems it should be `/var/lib/mysql`, without the `datadir` part:

```
root@mysql-85fbc78f68-lvx7h:/# ls /var/lib/mysql
auto.cnf  cachedb  ib_logfile0	ib_logfile1  ibdata1  lost+found  metadb  mlpipeline  mysql  performance_schema
```

this matches google search results

Reply:
Yes that is correct. It also works for me with /var/lib/mysql . Then upgrading might be easier.